### ISSUE TITLE
[BUG] [Linter] H007 (missing DOCTYPE) doesn't trigger after `{% load static %}`

### ISSUE THREAD

**Issue by graingert-coef** [djlint/djLint#1010]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/djlint/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info

- OS: Mac OS
- Python Version 3.12.7
- djLint Version 1.36.1
- template language: e.g. django 4.2 (LTS)

## Issue

when linting:

```django
{% load static %}
<html></html>
```
djlint doesn't say to add the missing doctype

## How To Reproduce

```
 thomas@Thomass-MacBook-Pro  ~/projects  cat demo.html
{% load static %}
<html></html>
 thomas@Thomass-MacBook-Pro  ~/projects  pipx run djlint --profile=django --include=H007 demo.html

Linting 1/1 files ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 00:00


Users/thomas/projects/demo.html
────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
H005 2:0 Html tag should have lang attribute. <html></html>
H016 2:0 Missing title tag in html. <html></html>
H020 2:0 Empty tag pair found. Consider removing. <html></html>
H030 2:0 Consider adding a meta description. <html></html>
H031 2:0 Consider adding meta keywords. <html></html>

Linted 1 file, found 5 errors.

 ✘  thomas@Thomass-MacBook-Pro  ~/projects  vim demo.html
 thomas@Thomass-MacBook-Pro  ~/projects  cat demo.html
<html></html>
 thomas@Thomass-MacBook-Pro  ~/projects  pipx run djlint --profile=django --include=H007 demo.html

Linting 1/1 files ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 00:00


Users/thomas/projects/demo.html
────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
H005 1:0 Html tag should have lang attribute. <html></html>
H007 1:0 <!DOCTYPE ... > should be present before the html tag. <html
H016 1:0 Missing title tag in html. <html></html>
H020 1:0 Empty tag pair found. Consider removing. <html></html>
H030 1:0 Consider adding a meta description. <html></html>
H031 1:0 Consider adding meta keywords. <html></html>

Linted 1 file, found 6 errors.
```

## Contents of djlint.toml/.djlintrc/pyproject.toml [tool.djlint]

empty
